spec/dpl/providers/pypi_spec.rb

```diff
@@ -37,16 +37,16 @@
     it { should have_run 'python setup.py other' }
   end
 
-  describe 'given --no-skip_upload_docs' do
-    it { should have_run 'python setup.py upload_docs --upload-dir build/docs -r https://upload.pypi.org/legacy/' }
+  describe 'given --skip_existing' do
```

Comment:
This doesn't look right/related to docs...

Reply:
Oh, you reordered the tests...

Reply:
oops, yes. sorry, i guess i did :)